### PR TITLE
Document volunteer/registration process

### PR DESCRIPTION
**Pull request by mxsasha**

### CODE REVIEW

**Comment by mxsasha** [comment]
Note that per https://trello.com/c/WC9aMCrr/212-evolve-volunteer-approaches we may want to evolve this process further - this is a first step in documenting what I have done so far, so that we have at least the existing process documented.

**Review by plaindocs — APPROVED** [review]
Looks great!

**Comment by mxsasha** [comment]
@alexfornuto I'd love to hear any input from you as well, as the volunteer coordinator for Portland. I'd want this to describe the process well for both Portland and Prague :)

**Comment by alexfornuto** [comment]
@mxsasha Cursory review looks good! I'll make a more thorough pass after my work day, but don't consider me a blocker if there's a time-crunch on this.

File: docs/organizer-guide/confs/registration.rst [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+* Red for vendors.
+* Green for temporary badges.
+
+Tasks of the on-site registration manager
+-----------------------------------------
+
+The on-site registration manager should be at the registration desk during
+the busiest times to solve any issues, and have access to Tito to look up
+registration details. During the night, they should move all attendee lists
+and badges to a locked room.
```
Comment (ericholscher) [inline docs/organizer-guide/confs/registration.rst]:
```suggestion
and badges to a locked room to preserve the privacy of attendees.
```

File: docs/organizer-guide/confs/registration.rst [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  blank badge.
+
+Regardless of the number of signs, people will often not queue up correctly
+by last name, so it may be needed to direct people to the right queue,
+especially during busy moments.
+
+If registration gets particularly busy, the on-site registration manager or
+floater can already hand out white or black lanyards to people in the queue,
+guide them to split up correctly, and provide other information like the venue
+layout.
```
Comment (ericholscher) [inline docs/organizer-guide/confs/registration.rst:110]: 💯 document. I wonder if there's an easy way to make this into a cheatsheet or checklist that we can actually print and put on the reg desk for each volunteer. Would be neat to have this for each role, but reg seems simplest to start.

File: docs/organizer-guide/confs/volunteer-coordination.rst [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+into several shifts that have one or two people assigned.
+
+Call for Volunteers
+-------------------
+
+The call for volunteers (CFV) typically starts when registration opens.
+About 2-3 months before the conference, the CFV closes, and volunteers
+are selected. Selection criteria include having a mix of volunteers with
+previous experience and those with no experience, and may also depend
+on the availability of volunteers.
```
Comment (ericholscher) [inline docs/organizer-guide/confs/volunteer-coordination.rst:28]: I wonder if we should note that we should try and pick volunteers prior to the tickets selling out. Its stressful for volunteers to not know if they have a volunteer ticket or not.

**Review by ericholscher — APPROVED** [review]
This is great. 👍

File: docs/organizer-guide/confs/volunteer-coordination.rst (hunk 3 above)
Comment (mxsasha) [reply docs/organizer-guide/confs/volunteer-coordination.rst:28]: Added 👍

**Comment by mxsasha** [comment]
@alexfornuto did you still want to review this document more?

**Pull request merged**